mac/hid.c

```diff
@@ -1176,6 +1176,17 @@ int HID_API_EXPORT_CALL hid_get_serial_number_string(hid_device *dev, wchar_t *s
 	return get_serial_number(dev->device_handle, string, maxlen);
 }
 
+int HID_API_EXPORT_CALL hid_darwin_get_location_id(hid_device *dev, uint32_t *location_id)
+{
+	int res = get_int_property(dev->device_handle, CFSTR(kIOHIDLocationIDKey));
```

Comment:
I'm pretty sure that the `location_id` is an unsigned int property
lets add a `get_uint_property` function, to be strictly correct

Reply:
Yes I also think it’s an unsigned property, and the return parameter is declared as such (`uint32_t *location_id`).

Since it’s the same bit size I don’t see any reason to add a separate getter. In fact `get_int_property()` is already used in `get_vendor_id()` to return an unsigned property the same way. But I added a cast just to be explicit.